route1io_connectors/google/sa360.py

```diff
@@ -292,8 +292,8 @@ def filter_zero_rows(df: pd.DataFrame, columns: list) -> pd.DataFrame:
     df = df.drop_duplicates()
     return df
 
-def sftp_upload_to_sa360(username: str, password: str, local_fpath: str,
-                         remote_fpath: str = None, known_hosts: str = None) -> None:
+def sftp_upload_to_sa360(username: str, password: str, known_hosts: str, local_fpath: str,
```

Comment:
Switched this to required arg to force users to use known_hosts file for security purposes